.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: deploy
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    # Ignore release tags
+    tags-ignore:
+      - hps-java-*
```

Comment:
Maybe I'm confused on what "deploy" means, but wouldn't we want maven to deploy _only_ on releases instead of everything else?

Reply:
> Maybe I'm confused on what "deploy" means, but wouldn't we want maven to deploy _only_ on releases instead of everything else?

It is Maven terminology - a deployment means uploading the snapshot (development) version of the jar whenever a merge is made to the master branch.

This is also done for jars produced by the release as part of a a separate workflow.